Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -107,8 +109,8 @@ namespace Hazel {
 		{
 			auto endTimepoint = std::chrono::high_resolution_clock::now();
```

Comment:
auto endTimepoint = std::chrono::steady_clock::now();

Reply:
Switched to steady_clock